src/Promitor.Core.Scraping/Configuration/Serialization/v1/Mapping/V1MappingProfile.cs

```diff
@@ -0,0 +1,48 @@
+﻿using AutoMapper;
+using Promitor.Core.Scraping.Configuration.Model;
+using Promitor.Core.Scraping.Configuration.Model.Metrics;
+using Promitor.Core.Scraping.Configuration.Model.Metrics.ResourceTypes;
+using Promitor.Core.Scraping.Configuration.Serialization.v1.Model;
+using Promitor.Core.Scraping.Configuration.Serialization.v1.Model.Metrics;
+using Promitor.Core.Scraping.Configuration.Serialization.v1.Model.Metrics.ResourceTypes;
+
+namespace Promitor.Core.Scraping.Configuration.Serialization.v1.Mapping
+{
+    public class V1MappingProfile : Profile
+    {
+        public V1MappingProfile()
+        {
+            CreateMap<MetricsDeclarationV1, MetricsDeclaration>();
+            CreateMap<AzureMetadataV1, AzureMetadata>();
+            CreateMap<MetricDefaultsV1, MetricDefaults>();
+            CreateMap<AggregationV1, Aggregation>();
+            CreateMap<ScrapingV1, Configuration.Model.Scraping>();
+            CreateMap<AzureMetricConfigurationV1, AzureMetricConfiguration>();
+            CreateMap<MetricAggregationV1, MetricAggregation>();
+            CreateMap<SecretV1, Secret>();
+
+            CreateMap<ContainerInstanceMetricDefinitionV1, ContainerInstanceMetricDefinition>();
```

Comment:
We might want to add this to https://github.com/tomkerkhove/promitor/blob/master/adding-a-new-scraper.md

Reply:
I was going to redo this after adding the v2 implementation. Are you ok with it being temporarily out of date, or do you want me to update it as part of this PR?

I don't mind either way, but obviously if I can avoid changing it and then changing it again shortly I'll do that 😸

Reply:
Sure, I'm ok with holding off for v2